subsys/net/lib/openthread/platform/radio.c

```diff
@@ -95,6 +96,10 @@ static struct k_work_q ot_work_q;
 static otError rx_result;
 static otError tx_result;
 
+#if defined(CONFIG_OPENTHREAD_TIME_SYNC)
+static otRadioIeInfo sTransmitIeInfo;
```

Comment:
nit: Zephyr does not use camel case naming afaik. However, i see two variables already sneaked in.